lib/cisco_node_utils/cmd_ref/feature.yaml

```diff
@@ -18,6 +18,11 @@ fabric_forwarding:
   config_get_token: '/^feature fabric forwarding$/'
   config_set: "feature fabric forwarding"
 
+fabric_frwd_anycast_mac:
+  config_get: "show running | i ^fabric"
+  config_get_token: '/^fabric forwarding anycast-gateway-mac (\S+)/'
+  config_set: "%s fabric forwarding anycast-gateway-mac %s"
```

Comment:
@smigopal @chrisvanheuveln This was my fault.  Smitha pinged me that there was another feature that needed to be enabled in addition to `feature fabric forwarding` and I suggested she add that to the feature object as well.  I did not realize it had a variable component to it.  I agree that we need a separate provider for this piece so the user can set the correct value.